config/locales/en.yml

```diff
@@ -37,6 +37,16 @@ en:
               inclusion: '%{value} is not a valid relationship for related works'
   errors:
     not_authorized: 'You are not authorized to perform the requested action.'
+  publication_date:
+    edit:
+      legend: 'Previous publication date'
+      fields:
+        year:
+          label: 'Year'
+        month:
+          label: 'Month'
+        day:
+          label: 'Day'
```

Comment:
Trying to insulate nested fields from needing to care about where they're nested.